### DISCUSSION TITLE
Add ability to select specific jobs to run

### DISCUSSION THREAD

**Discussion by Jarod1662** [community/community#58050] [Actions]
### Select Topic Area

Product Feedback

### Body

Please add the ability to select a specific job to run before we trigger the workflow.

We are currently migrating from DevOps to GitHub actions and sometimes we use the stage selection to specify what jobs to run; for example: we have a pipeline that deploys several azure resources however we have made changes to 2 of the services. we should be able to go into Actions and select what jobs to run for example: deploy WebApp or Storage account but not the rest.

Stage select in DevOps:
![image](https://github.com/community/community/assets/35074371/085d5495-f6ff-4bd1-9ef2-3ee7dbe950d0)

**Comment by jshtuazon2** [comment]
https://github.com/orgs/community/discussions/58050#discussion-5298612
